hypha/apply/funds/views.py

```diff
@@ -90,6 +90,33 @@
 )
 
 
+class UpdateReviewerActions:
```

Comment:
Is this the best way to do this? I want to avoid having the same logic in two places.

Reply:
Looks good, we can name it a Mixin say UpdateReviewersMixin to convey it is to be used by other views. Also, later common methods can be included in it.

Reply:
Renamed, thanks!